packages/admin-ui-components/webpack.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       'node_modules',
       path.join(__dirname, 'src/fonts'),
     ],
-    extensions: [".ts", ".tsx", ".js", ".jsx", ".less"],
+    extensions: [".ts", ".tsx", ".js", ".jsx", ".less", ".scss"],
```

Comment:
why did we need to add this now?

Reply:
hm, this change looks useless, will revert it.